examples/mcts_example.py

```diff
@@ -0,0 +1,396 @@
+        pass
+
+
+class ActionNode(Node):
+
+    def __init__(self, game, hash_key, opp=False):
+        super().__init__()
+        self.hash_key = hash_key
+        self.available_actions = self._extract_actions(game)
+        self.children: List[ChangeNode] = []
+        self._opp = opp
+        self.terminal = game.state.game_over
+
+    def opp(self):
+        return self._opp
+
+    def _extract_actions(self, game):
+        actions = []
+        for action_choice in game.get_available_actions():
+            if action_choice.action_type == botbowl.ActionType.PLACE_PLAYER:
+                continue
+            if action_choice.action_type == botbowl.ActionType.END_SETUP:
+                continue
+            if len(action_choice.players) > 0:
+                for player in action_choice.players:
+                    actions.append(Action(action_choice.action_type, position=None, player=player))
+            elif len(action_choice.positions) > 0:
+                for position in action_choice.positions:
+                    actions.append(Action(action_choice.action_type, position=position))
+            else:
+                actions.append(Action(action_choice.action_type))
+        return actions
+
+    def is_fully_expanded(self):
+        return len(self.children) == len(self.available_actions)
+
+    def print(self, tabs=0):
+        t = ''.join(['\t' for _ in range(tabs)])
+        min_max = 'min' if self.opp() else 'max'
+        if len(self.children) == 0:
+            print(f"{t}<ActionNode p='{min_max}' visits={self.num_visits()} score={self.score()} actions={len(self.available_actions)}/>")
+        else:
+            print(f"{t}<ActionNode p='{min_max}' visits={self.num_visits()} score={self.score()} actions={len(self.available_actions)}>")
+            for child in self.children:
+                child.print(tabs+1)
+        if len(self.children) > 0:
+            print(f'{t}</ActionNode>')
+
+    def depth_stats(self):
+        if len(self.children) == 0:
+            return 0, 0, 0
+        min_depths = []
+        max_depths = []
+        avg_depths = []
+        for child in self.children:
+            min_d, max_d, avg_d = child.depth_stats()
+            min_depths.append(min_d)
+            max_depths.append(max_d)
+            avg_depths.append(avg_d)
+        return 1 + np.min(min_depths), 1 + np.max(max_depths), 1 + np.mean(avg_depths)
+
+
+class ChangeNode(Node):
+
+    def __init__(self, parent, action):
+        super().__init__()
+        self.parent = parent
+        self.action = action
+        self.outcomes = {}  # hash: Node
+        self.evaluations = []
+        self.terminal = False
+
+    def print(self, tabs=0):
+        t = ''.join(['\t' for _ in range(tabs)])
+        min_max = 'min' if self.parent.opp() else 'max'
+        if len(self.outcomes) == 0:
+            print(f"{t}<ChanceNode p='{min_max}' visits={self.num_visits()} score={self.score()} action='{self.action.to_json()}'/>")
```

Comment:
```suggestion
            print(f"{t}<ChanceNode p='{min_max}' visits={self.num_visits()} score={self.score()} action='{self.action}'/>")
```
More pythonic to use the defined __repr__ function  =)